catalog/src/table_operator.rs

```diff
@@ -74,6 +77,12 @@ impl TableOperator {
 
             match table_result {
                 Ok(Some(table)) => {
+                    // When table open successfully, try flush to reduce WAL size, so when reopen
+                    // this shard the WAL required to fetch is reduced.
+                    if let Err(e) = table.flush(FlushRequest { sync: false }).await {
+                        warn!("Try flush after open table failed, err:{e}");
+                    }
```

Comment:
If we report shard has opened before flush finished, maybe it will have no effect about reduce wal size in retry.
It is possible that, the retry request will be executed before none of the flush works is finished.